src/components/Home.vue

```diff
@@ -41,8 +41,8 @@
         </v-list-item-group>
       </v-list>
     </v-navigation-drawer>
-    <router-view />
-    <v-footer
+    <router-view v-bind:getTaskPath="getTaskPath"/>
```

Comment:
I recommend renaming the attribute and value `taskPath` since HTML attributes and Vue properties are usually named using nouns.